scripts/data_transfer/copy_uploaded_CellRegMap_input_files.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -ex
+
+gsutil mv -r "gs://cpg-tob-wgs-main-upload/CellRegMap_input_files/" gs://cpg-tob-wgs-main/scrna-seq/
```

Comment:
I don't think `-r` exists for `mv`, but you'll want to use `-m` to use multi-threaded operations.

```suggestion
gsutil mv -m "gs://cpg-tob-wgs-main-upload/CellRegMap_input_files/" gs://cpg-tob-wgs-main/scrna-seq/
```